hasura/migrations/default/1690929977427_alter_table_public_circles_add_column_starting_tokens/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."circles" add column "starting_tokens" integer
```

Comment:
the date on this migration is now old, and we have cosoul migrations that cannot be undone with newer dates, so can you just rename this date time to be the newest again, or create a new migration and put this in there?